submissions/medinatapanpa/landingPage/dufuna.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
Fix indentation on this file. Refer to STYLEGUIDE.md for more information about indentation